Resources/public/js/auth.js

```diff
@@ -1,107 +1,136 @@
-'use strict';
+import u2fApi from 'u2f-api'
```

Comment:
This would force people to use some sort of bundler (like webpack) to have cross-browser support.

Reply:
That's intentionally, but opinionated ;). I assume users of this bundle use Symfony Encore, but I'm not sure if that's a good idea.

How to create a JS file that can be used with and without Webpack?

Reply:
I agree 😉. 

> I've added a package.json file, but that doesn't make any sense right now. How to solve this?

Just add a section in the readme on how to add `u2f-api`.

> How to create a JS file that can be used with and without Webpack?

The `auth` object from `auth.js` should be exported (Backwards compatible ES6 export example: https://github.com/dandv/local-iso-dt/blob/master/index.js). Then we could add a section to the readme on how to include them:

**es6-style**
```js
import u2fApi from 'u2f-api'
import auth from './auth'

auth(u2fApi);
```

**es5-style**
```js
<script src=".../u2f-api.js"></script>
<script src=".../auth.js"></script>
<script>
auth(u2fApi());
</script>
```
Just a rough idea. Maybe there are better ways.